op-node/rollup/derive/engine_queue.go

```diff
@@ -603,6 +655,9 @@ func (eq *EngineQueue) forceNextSafeAttributes(ctx context.Context) error {
 }
 
 func (eq *EngineQueue) StartPayload(ctx context.Context, parent eth.L2BlockRef, attrs *eth.PayloadAttributes, updateSafe bool) (errType BlockInsertionErrType, err error) {
+	if eq.isEngineSyncing() {
+		return BlockInsertTemporaryErr, fmt.Errorf("engine is in progess of unsafe sync")
```

Comment:
Engine will be always trying to run unsafe sync. In this case, we need to early return when unsafe block is advancing to enginesynctarget. So I think 
```suggestion
		return BlockInsertTemporaryErr, fmt.Errorf("unsafe block advancing to reach engineSyncTarget")
```
will be more descriptive.